packages/local-notifications/index.ios.ts

```diff
@@ -204,25 +224,26 @@ export class LocalNotificationsImpl extends LocalNotificationsCommon implements
 							// Just fall back to a normal notification...
 						}
 					}
-					triggers.forEach(trigger => registerNotification(trigger[0], content, trigger[1], scheduledIds))
 				});
 			}
+
+			registerNotification(entry.id, content, trigger, scheduledIds);
+
+			if (interval && entry.displayImmediately) {
+				const id = LocalNotificationsImpl.generateNotificationID();
```

Comment:
Unlike the old logic, won't this generate a new ID. What will happen if I decide to cancel the ID of the original request? To me the old logic would cancel both the immediately scheduled one and the other one that was scheduled with interval.

Reply:
I don't think it worked for you that way. Perhaps, you're confused.
Even now, immediate notifications have their own unique ID.
See:
https://github.com/NativeScript/plugins/blob/main/packages/local-notifications/index.android.ts#L241
https://github.com/NativeScript/plugins/blob/main/packages/local-notifications/index.ios.ts#L151